internal/cloudplugin/interface.go

```diff
@@ -0,0 +1,9 @@
+package cloudplugin
+
+import (
+	"io"
+)
+
+type Cloud1 interface {
```

Comment:
Did you name this `Cloud1` because there will eventually be a Cloud2,3 etc? Can you share more about this naming?

Reply:
Yes, it's a preemptive version number to ensure good organization later on when we have to break the protocol later on and add a version 2.

Reply:
go-plugin will automatically do version negotiation to select the latest version that a server (plugin) provides.